Sources/Prometheus/MetricTypes/Gauge.swift

```diff
@@ -67,15 +69,15 @@ public class Gauge<NumType: Numeric, Labels: MetricLabels>: Metric, PrometheusHa
     ///     - amount: Amount to set the gauge to
     ///     - labels: Labels to attach to the value
     ///
-    public func set(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
+    public func set(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping () -> Void = { }) {
```

Comment:
It's not though, since it's a gauge we're setting it to whatever value we pass in, right? What do you think about keeping this similar to the API for `Counter` vs. simplifying the call?

Reply:
Yeah, my bad, I saw gauge and my head defaulted to `Gauge.inc` and `Gauge.dec`. You can disregard my comment :)